sunpycube/cube/NDCube.py

```diff
@@ -71,12 +69,8 @@ def plot(self, *args, **kwargs):
     def __getitem__(self, item):
         if item is None or (isinstance(item, tuple) and None in item):
             raise IndexError("None indices not supported")
-        return self._getitem_NDCube(item)
-
-    def _getitem_NDCube(self, item):
         data = self.data[item]
-        # wcs = self.wcs.wcs_slicer[item]
-        wcs = None
+        wcs = self.wcs.wcs_slicer[item]
```

Comment:
Any thoughts on allowing the user to input an ```astropy.wcs``` object and have ```NDCube.__init__``` convert it to a ```sunpycube.WCS```?  @Cadair @abit2?